StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetError.swift

```diff
@@ -53,8 +53,7 @@ public enum PaymentSheetError: Error {
     case failedToCreateLinkSession
     case linkNotAuthorized
 
-    /// Localized description of the error
-    public var localizedDescription: String {
```

Comment:
👋 To add context on why this change is backwards-compatible:

If the error type is preserved as a `PaymentSheetError` (e.g. `(error as? PaymentSheetError).localizedDescription`) returns the same value. 

But this fixes getting the localized desc when not casting the error type (e.g. `(error as Error).localizedDescription` will now return NSError.stp_unexpectedErrorMessage() whereas before it did not.